### DISCUSSION TITLE
GitHubの使用状況は、どんな感じです、、、？

### DISCUSSION THREAD

**Discussion by kazzyfrog** [lgtm-factory/lgtm-factory#17] [投票]
ちなみに、現状のGitHubの使用状況として、どんな感じです、、、？

全員で、助け合いながら、協力するためにも、
GitHub活用のレベル感？などを、みんなでざっくり把握できればと思いますー！！

**Comment by kazzyfrog** [comment]
Discussionsのテストも兼ねて、お手隙の際に投票してくださいーー👍

**Comment by kazzyfrog** [comment]
あれ、間違えた、投票匿名？にしてしまったかも。。

**Reply by hamanyann** [reply]
絵文字の左の上矢印はなんですか

**Comment by hamanyann** [comment]
![スクリーンショット 2024-08-05 211957](https://github.com/user-attachments/assets/89020ce5-de7b-4f0c-8c9c-29586b6a67df)

**Reply by kazzyfrog** [reply]
これは、[賛成票](https://docs.github.com/ja/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-discussion)という機能です！

いろんな、ディスカッションのスレッドが作成されていたり、
このスレッド内で、たくさんのコメントが書き込まれたりした場合に、
マーキングできる「いいね」的な機能です！

数字が出るので、どのコメントや、スレッドが人気か、というのが見える感じです！👍

**Reply by kagomen** [reply]
知らなかった！👀

**Reply by ghost** [reply]
はえーーーーー.....✍️
